screenpy/actions/see_all_of.py

```diff
@@ -51,10 +51,19 @@ def describe(self: SelfSeeAllOf) -> str:
     @beat("{} sees if {log_message}:")
     def perform_as(self: SelfSeeAllOf, the_actor: Actor) -> None:
         """Direct the Actor to make a series of observations."""
+        all_passed = True
         for question, resolution in self.tests:
-            the_actor.should(See.the(question, resolution))
+            try:
+                the_actor.should(See.the(question, resolution))
+            except AssertionError:
+                all_passed = False
+
+        if not all_passed:
+            raise AssertionError(f"{the_actor} did not find all expected answers!")
```

Comment:
This feels like it makes using `SeeAllOf` more complicated when a failure occurs.  
When one of tests fails, the exception that occurs will now be obfuscated by this new assertion.  This would be especially true when now multiple errors might occur.

Reply:
Oh yes, woops. We should be tracking the failures and raise them all.

Reply:
Raise them all or print the stack for each?